modular_tannhauser/modules/CitOwOChems/code/chemistry/recipes/OwOCookBook.dm

```diff
@@ -2,100 +2,25 @@
 	var/name = null
 	var/id = null
 
-
```

Comment:
So none of this block was working?

Reply:
OwOExplode was never called by anything, so I've no idea if the code itself works. I could work to implement it, though it looks like it's just a fancy failed reaction.